### DISCUSSION TITLE
Cannot preview or commit edits

### DISCUSSION THREAD

**Discussion by budryerson** [community/community#22466] [New to GitHub]
<p>I’ve been editing files on GitHub for years.  Suddenly, I cannot preview or ‘commit’ an edit.  (Also, I cannot name a successor in my settings.)  I think there are other things wrong, too; but something has changed and I would dearly like to get it fixed.  I’ve been beating my head against this for hours.</p>

**Comment by jfullstackdev** [comment]
<p>there is another one also with the same situation as yours,</p>
<p>is the problem still present ?</p>

**Comment by vkenn** [comment]
<p>I have had the same problem too</p>

**Comment by vkenn** [comment]
<p>How did you solve it ?</p>

**Comment by budryerson** [comment]
<p>Yes, the problem is still present. I have not received any help, either. :frowning_face:<br>
(Sorry.  They instructed me to use emojis.)</p>
<p>The problem seems systemic.</p>
<ol>
<li>It appeared suddenly for no apparent reason</li>
<li>I also am unable to do other things on GitHub.</li>
</ol>
<p>I have been all over the blog, and docs, and everywhere.  I wish I could get some help.</p>

**Comment by jsoref** [comment]
<p>I’d contact:<br>
<a href="https://support.github.com" class="onebox" target="_blank" rel="noopener">https://support.github.com</a></p>

**Comment by budryerson** [comment]
<p>That’s the first thing I tried.  It simply says to use the file editor and, if that doesn’t work, which it doesn’t, to “contact support,” which takes me right back to where I began.  I have tried stating the problem every way I can think of.  It’s probably something obvious, something got turned on or off by mistake, and I don’t know what that is.  I dearly wish someone could look at the damn thing and say, “Oh!  Your status merge push commit signature is on verify.  Why did you do that?”  That would be wonderful.</p>

**Comment by jsoref** [comment]
<p>Try:</p>
<p><a href="https://support.github.com/contact/bug-report" class="onebox" target="_blank" rel="noopener">https://support.github.com/contact/bug-report</a></p>
<p>(They recently changed the support web site’s flow to make it harder to get to a form.)</p>

**Comment by budryerson** [comment]
<p>Thank you so much.  I “created a ticket.”  Now, let’s see what happens.<br>
You are correct that they made themselves hard to find.  Thank you again.</p>
<p>Bud Ryerson<br>
San Francisco</p>

**Comment by budryerson** [comment, marked as answer]
<p>I use Firefox as my browser.  My editing problem turns out to involve Ghostery, which is a fairly common browser extension used to block trackers and ads.  By turning off the extension or browsing in a private window, that fixes the problem.  But the best solution for me was adding “<a href="http://github.com" rel="noopener nofollow ugc">github.com</a>” to the extension as a “trusted site.”</p>
<p>I can’t say whether GitHub changed recently or Ghostery did, but GitHub support could be easier to reach.  I was all over the GitHub universe trying to find help.  Whatever questions I posted to the support page always directed me to some inane and extremely unhelpful disquisition.  Finally, I type in the right question, “How do I contact support?” and that led me to “Create a ticket,” which is how I finally got a thoughtful, helpful, and human response.</p>
<p>Anyway, thanks to the folks at GitHub support, this problem appears to be solved for now.  I hope that this solution works for you, too.</p>
<p>Bud Ryerson<br>
San Francisco</p>

**Comment by budryerson** [comment]
<p>Firefox extension called Ghostery.  Add <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> as a trusted site.</p>

**Comment by budryerson** [comment]
<p>Firefox extension called Ghostery. Add <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> as a trusted site.</p>
